src/screens/FundCampaignPledge/FundCampaignPledge.tsx

```
@@ -215,6 +213,16 @@
     setAnchor(anchor ? null : event.currentTarget);
   };
 
+  const isWithinCampaignDates = useMemo(() => {
+    if (!pledgeData?.fundCampaign) return false;
+
+    const now = dayjs();
+    let start = dayjs(pledgeData.fundCampaign.startAt);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'start' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)